### ISSUE TITLE
[Inlets] Inlets guide points to non-existent releases page

### ISSUE THREAD

**Issue by radu-matei** [deislabs/krustlet-docs#7]
The Inlets guide points to the project releases page to download a pre-buit binary for Inlets.
That page does not contain any releases, so we should either update it with the appropriate page, or remove the reference.

cc @alexellis

**Comment by alexellis** [comment]
Hi @radu-matei the binaries now exist here: https://github.com/inlets/inlets-pro/releases

The guide needs a couple of tweaks as inlets is now a paid product, but there is a monthly tier with a lower price for personal users.

I'd be happy to take a look at updating this, it's a relatively trivial change that's needed.

Alex

**Comment by radu-matei** [comment]
Is there still the option to build your own binaries?

Regardless, I personally think we should update the guide and definitely keep it, as it is a very helpful addition, but I defer to @thomastaylor312 and / or @bacongobbler.

Thanks!

**Comment by thomastaylor312** [comment]
Yeah, I think it is still useful to have! I would just mention in the doc update that it is a paid tool so it doesn't catch anyone by surprise
